plugin.php

```diff
@@ -489,12 +494,14 @@ protected static function register_get($content) {
                        "Full name or company name*",
                        "Email address*",
                        "Job title or company field*",
+                       "Website | Filmography",
                        "Country*",
                        "City",
-                       "Website | Filmography",
                        "Individual or organisation?",
                        "Individual",
                        "Organisation / Company",
+                       "Help us verify your signature",
+                       "If the submitted information does not already show it, please give us some hints on your connection to the film and television industry.",
```

Comment:
This might be a data privacy issue if registering people name other people here who have not consented to their data being stored here.